action.js

```diff
@@ -126,31 +132,32 @@ module.exports.getBodyFromGitHubContext = getBodyFromGitHubContext;
 // the user wishes us to do a best-effort fill of the events body using all the
 // information available in the GitHub context object.
 //
-const constructBody = ( change_id,
-                        custom,
-                        github,
-                        pipeline_id,
-                        stage,
-                        stage_ref,
-                        status,
-                        team_id ) => {
-  //
+const constructBody = ( github,
+                        team_id,
+                        event_name,
+                        event_action,
+                        repo,
+                        environment,
+                        branch,
+                        commit,
+                        image,
+                        custom
```

Comment:
based on the payload that Brian included, it should be `meta` not `custom`

Reply:
Yup.